app/views/languages/_form.html.erb

```diff
@@ -0,0 +1,4 @@
+<%= form_for @language do |f| %>
+  <%= f.text_field :name, placeholder: 'Name' %>
```

Comment:
Could you make this placeholder text a bit more descriptive by changing it to something like "Language"?